readthedocs/api/v2/views/integrations.py

```diff
@@ -271,6 +289,21 @@ def handle_webhook(self):
                 raise ParseError('Parameter "ref" is required')
         if event in (GITHUB_CREATE, GITHUB_DELETE):
             return self.sync_versions(self.project)
+
+        if (
+            event == GITHUB_PULL_REQUEST and
+            self.data['action'] in [GITHUB_PULL_REQUEST_OPEN, GITHUB_PULL_REQUEST_SYNC]
```

Comment:
This will handle both PR `opened` and `synchronized` (new commit in the PR) events.

Reply:
Tested it locally and it worked great 👍